remix-solidity/src/compiler/compiler.js

```diff
@@ -29,6 +29,8 @@ function Compiler (handleImportCall) {
 
   var evmVersion = null
 
+  var language = 'Solidity'
```

Comment:
You can use `let` here.

Reply:
yes you're right but i'd prefer to let this TODO for the typescript rewrite (all the other var are `var`...)